android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,5 +1,5 @@
-• Android Auto public preview
-• Temporarily disabled OpenStreetMap login to pass the Google Play review (see https://github.com/organicmaps/organicmaps/issues/7000)
+• Android Auto
+• Temporarily disabled OpenStreetMap login to pass Google Play review https://github.com/organicmaps/organicmaps/issues/7000
```

Comment:
This is important for Google Play, but is not important for FDroid. @pastk can you please help, and create two commits:
1. First one with ALL these changes.
2. The second one that removes this "Temporarily..." line in all languages.

Reply:
Does F-Droid take release notes from `android/app/src/fdroid/play/release-notes/en-US/default.txt` or from `android/app/src/fdroid/play/listings/en-US/release-notes.txt`?
@rtsisyk

Reply:
ah, ok, the first symlinks to the latter

Reply:
done